dotnet/src/Functions/Functions.OpenApi/Model/RestApiSecurityScheme.cs

```diff
@@ -0,0 +1,65 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Diagnostics.CodeAnalysis;
+
+namespace Microsoft.SemanticKernel.Plugins.OpenApi;
+
+/// <summary>
+/// REST API Security Scheme.
+/// </summary>
+[Experimental("SKEXP0040")]
+public sealed class RestApiSecurityScheme
+{
+    /// <summary>
+    /// REQUIRED. The type of the security scheme. Valid values are "apiKey", "http", "oauth2", "openIdConnect".
+    /// </summary>
+    public string SecuritySchemeType { get; init; }
+
+    /// <summary>
+    /// A short description for security scheme. CommonMark syntax MAY be used for rich text representation.
+    /// </summary>
+    public string? Description { get; init; }
+
+    /// <summary>
+    /// REQUIRED. The name of the header, query or cookie parameter to be used.
+    /// </summary>
+    public string Name { get; init; }
+
+    /// <summary>
+    /// REQUIRED. The location of the API key. Valid values are "query", "header" or "cookie".
+    /// </summary>
+    public string ParameterLocation { get; init; }
```

Comment:
SK has a model class for parameter location - RestApiOperationParameterLocation.